examples/gradle/dokka-multiplatform-example/build.gradle.kts

```diff
@@ -19,7 +19,7 @@ kotlin {
     jvm() // Create a JVM target with the default name "jvm"
     linuxX64("linux")
     macosX64("macos")
-    js()
+    js(BOTH)
```

Comment:
`js()` is considered to be legacy and throws an error. 

```
* What went wrong:

==========
This project currently uses the Kotlin/JS Legacy compiler backend, which has been deprecated and will be removed in a future release.
Please migrate the project to the new IR-based compiler (https://kotl.in/jsir) by adding:
- kotlin.js.compiler=ir to your gradle.properties file.
- js(IR) { ... } to your build file.
You can continue to use the deprecated legacy compiler in the current version of the toolchain by specifying it explicitly.
==========
```

However, it seems like you can still use the legacy backend, so I decided to go with `BOTH` to check for backwards compatibility in this and future releases. I don't think everyone will migrate at once, so it's worth giving some time. We can consider changing it to `IR` if/when it starts causing problems.